### PR TITLE
Delete

### PR DESCRIPTION
**Pull request by JasonPlatzer**
fixes problem with other pull request

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -36,6 +37,17 @@ def add_book():
...
     
 
+def delete_book():  
+    try:
+        books = store.get_all_books()
+        ui.show_books(books)
+        book_id = ui.get_book_id()
+        book = store.get_book_by_id(book_id)
+        book.delete()
+    except UnboundLocalError:
```
Comment (claraj) [inline main.py:47]: Don't catch this error  - UnboundLocalError means there's a bug somewhere else in the code.  Can you find and fix? (Hint - another issue asks you to fix the bug)

**Comment by JasonPlatzer** [comment]
fixed issue

**Comment by claraj** [comment]
"fixes problem with other pull request" Which pull request? Link to the other one.
"fixed issue" you need to be more specific describing what your changes are.

File: bookstore.py [hunk 2]
```diff
@@ -192,7 +192,7 @@ def get_book_by_id(self, id):
             
             if book_data:
                 book = Book(book_data['title'], book_data['author'], book_data['read'], book_data['rowid'])
-                    
+            book = None
```
Comment (claraj) [inline bookstore.py:195]: This is a fix for issue #5 so should be in that PR

File: main.py [hunk 3]
```diff
@@ -36,6 +37,17 @@ def add_book():
...
     
 
+def delete_book():  
+    try:
+        books = store.get_all_books()
+        ui.show_books(books)
+        book_id = ui.get_book_id()
+        book = store.get_book_by_id(book_id)
+        book.delete()
+    except BookError:
```
Comment (claraj) [inline main.py:47]: This is better than catching UnboundLocalError

**Pull request merged**